[2015-03-17 09:55:40.412] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 09:55:41.336] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 10:27:29.038] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 10:27:30.441] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 11:42:03.762] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 11:42:05.437] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:00:30.047] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:00:31.294] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:00:47.956] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:00:49.275] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:07:28.011] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:07:29.279] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:21:16.566] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:21:18.147] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:24:53.455] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:24:54.625] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:26:09.860] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:26:11.037] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:26:31.644] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:26:33.396] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:26:49.342] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:26:51.169] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:27:14.863] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:27:15.989] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:27:29.468] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:27:30.813] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:27:50.520] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:27:51.803] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:29:25.306] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:29:26.750] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:34:14.444] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:34:15.663] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:36:32.523] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:36:33.858] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:37:06.855] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:37:08.572] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:38:19.440] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:38:20.644] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:41:45.084] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:41:46.236] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:42:10.816] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:42:12.185] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:42:45.550] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:42:46.906] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:44:17.692] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:44:19.296] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:48:20.755] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:48:21.844] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:49:40.695] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:49:42.061] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:50:00.773] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:50:02.009] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:50:38.413] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:50:39.513] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:51:33.318] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:51:34.534] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:51:59.042] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 12:52:00.245] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:16:12.162] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:16:13.554] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:18:12.229] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:18:13.413] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:21:20.326] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:21:21.934] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:21:46.145] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:21:46.989] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:22:51.372] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:22:52.202] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:24:39.090] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:24:40.435] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:29:20.117] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:29:21.584] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:35:39.910] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:35:41.392] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:36:05.109] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:36:06.424] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:36:31.274] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:36:32.772] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:38:12.434] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:38:13.877] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:39:35.603] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:39:36.698] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:39:43.917] [ERROR] normal - [TypeError: Object #<Answer> has no method 'dbInfo']
TypeError: Object #<Answer> has no method 'dbInfo'
    at Q.all.spread.then.then.then.then.then.then.then.done.self.json.err (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:44:28)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 13:42:09.074] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:42:10.202] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:43:53.811] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:43:55.130] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:44:17.049] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:44:18.567] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:44:38.318] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:44:39.612] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:45:40.434] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:45:41.567] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:46:23.590] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:46:25.202] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:47:20.158] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:47:21.418] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:52:38.886] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:52:39.847] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:53:06.210] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:53:07.405] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:53:43.919] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:53:45.482] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:54:08.086] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:54:09.513] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:58:06.149] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 13:58:07.810] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 14:35:13.778] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 14:35:15.469] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 14:40:42.528] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 14:40:44.291] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 14:42:44.096] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 14:42:45.943] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:06:40.028] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:06:42.074] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:08:23.813] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:08:25.265] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:08:43.125] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:08:44.818] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:09:16.148] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:09:17.996] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:11:36.123] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:11:38.110] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:12:12.256] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:12:13.662] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:13:39.666] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:13:40.738] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:14:49.580] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:14:50.516] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:15:42.531] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:15:43.996] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:16:20.118] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:16:21.667] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:17:29.437] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:17:30.722] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:20:11.635] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:20:12.415] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:25:22.897] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:25:24.002] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:40:56.667] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:40:58.079] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:41:52.490] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:41:53.550] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:42:51.624] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:42:52.648] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:46:48.638] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:46:50.266] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:55:02.264] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:55:03.575] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:57:13.522] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:57:13.782] [ERROR] normal - [ReferenceError: custpwdrest is not defined]
ReferenceError: custpwdrest is not defined
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:149:39
    at Object.<anonymous> (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:151:2)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at d:\Workspaces\node\ccorolla\lib\core\$include.js:56:21
    at Array.forEach (native)
[2015-03-17 15:57:57.405] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:57:59.114] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:58:11.392] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 15:58:12.596] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:10:14.212] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:10:15.541] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:13:26.331] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:13:28.134] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:13:35.702] [ERROR] normal - [Error: tel not input!]
Error: tel not input!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:123:30)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:13:35.710] [ERROR] normal - [TypeError: Cannot call method 'failure' of undefined]
TypeError: Cannot call method 'failure' of undefined
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:139:23
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:24:25.190] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:24:26.622] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:24:36.399] [ERROR] normal - [Error: tel not input!]
Error: tel not input!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:123:30)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:24:36.406] [ERROR] normal - [TypeError: Cannot call method 'failure' of undefined]
TypeError: Cannot call method 'failure' of undefined
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:141:23
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:27:31.724] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:27:33.137] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:27:35.440] [ERROR] normal - [Error: tel not input!]
Error: tel not input!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:123:30)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:28:30.684] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:28:32.340] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:33:47.883] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:33:49.014] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:44:34.505] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:44:35.975] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:44:54.902] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:44:56.258] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:45:35.398] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:45:36.891] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:47:04.044] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:47:05.437] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:47:52.728] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:47:53.526] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:47:59.203] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:89:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:49:09.344] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:49:10.707] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:49:14.223] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:89:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:50:42.302] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:50:43.402] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:51:04.574] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:51:05.851] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:51:07.267] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:89:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:51:51.771] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:51:53.545] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:55:52.245] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:55:53.892] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:55:56.612] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:89:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:55:58.898] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:89:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:56:00.009] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:89:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:57:10.636] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:57:11.932] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 16:57:17.092] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:89:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 16:57:19.156] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:89:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:03:20.261] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 17:03:22.146] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 17:03:26.672] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:89:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:03:43.048] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:89:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:06:09.655] [ERROR] normal - [Error: tel not input!]
Error: tel not input!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:127:30)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:07:08.494] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:34:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:07:14.510] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:34:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:09:06.071] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 17:09:07.143] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 17:09:13.371] [ERROR] normal - [TypeError: Cannot read property 'TResult' of undefined]
TypeError: Cannot read property 'TResult' of undefined
    at Answer.json (d:\Workspaces\node\ccorolla\lib\response\answer.js:161:48)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:68:22
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\Q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:09:14.502] [ERROR] normal - [TypeError: Cannot read property 'TResult' of undefined]
TypeError: Cannot read property 'TResult' of undefined
    at Answer.json (d:\Workspaces\node\ccorolla\lib\response\answer.js:161:48)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:68:22
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\Q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:09:19.077] [ERROR] normal - [TypeError: Cannot read property 'TResult' of undefined]
TypeError: Cannot read property 'TResult' of undefined
    at Answer.json (d:\Workspaces\node\ccorolla\lib\response\answer.js:161:48)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:68:22
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\Q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:09:19.794] [ERROR] normal - [TypeError: Cannot read property 'TResult' of undefined]
TypeError: Cannot read property 'TResult' of undefined
    at Answer.json (d:\Workspaces\node\ccorolla\lib\response\answer.js:161:48)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:68:22
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\Q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:09:34.860] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 17:09:36.222] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 17:09:41.603] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:90:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 17:09:44.123] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:90:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 18:47:35.434] [ERROR] normal - checked redis connect is error!
[2015-03-17 18:47:35.451] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-17 18:47:45.136] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:47:46.696] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:50:27.586] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:50:28.693] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:50:37.405] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:50:38.775] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:51:37.205] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:51:38.552] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:52:01.930] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:52:03.008] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:52:16.941] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:92:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 18:56:35.070] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 18:56:36.419] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:02:30.774] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:02:32.476] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:06:45.252] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:06:46.413] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:11:54.259] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:11:55.302] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:12:39.722] [ERROR] normal - [TypeError: Cannot read property 'CUST_ID' of null]
TypeError: Cannot read property 'CUST_ID' of null
    at Object.cust2user (d:\Workspaces\node\ccorolla\lib\biz\customer.js:41:13)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:107:41
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 19:15:12.778] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:15:14.410] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:15:22.974] [ERROR] normal - [TypeError: Cannot read property 'CUST_ID' of null]
TypeError: Cannot read property 'CUST_ID' of null
    at Object.cust2user (d:\Workspaces\node\ccorolla\lib\biz\customer.js:42:13)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:107:41
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 19:16:29.774] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:16:31.265] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:24:47.501] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:24:48.752] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:26:20.190] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:26:21.645] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:26:57.609] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:26:59.204] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:27:48.500] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:91:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 19:28:39.631] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:91:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 19:33:12.128] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:33:14.667] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:34:39.603] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:34:41.405] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:37:35.133] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:37:36.284] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:38:30.843] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:38:32.072] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:39:45.419] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:39:46.876] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:40:29.427] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:40:31.227] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:40:56.968] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:40:58.006] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:42:06.440] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:42:07.872] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:56:22.260] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:56:23.542] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:56:29.715] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:35:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 19:57:18.397] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 19:57:19.885] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:07:40.516] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:07:42.105] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:08:23.064] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:08:24.872] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:12:22.060] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:12:23.743] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:12:29.445] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:91:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 20:21:09.967] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:21:11.312] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:22:33.870] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:22:35.765] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:25:37.065] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:25:38.889] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:31:32.836] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:31:37.317] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:35:21.826] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:35:22.991] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:37:17.766] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:37:22.537] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:38:06.802] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:38:11.983] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:38:26.468] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:38:30.222] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:38:37.045] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:38:38.665] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:38:49.478] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:38:53.200] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:39:21.728] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:39:23.517] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:39:36.323] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:91:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 20:40:49.332] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:40:50.790] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:41:38.171] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:41:39.342] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:43:02.901] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:43:04.740] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:51:29.014] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:51:30.795] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:52:29.471] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:52:31.156] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:53:13.474] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:53:14.653] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:53:55.519] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:53:57.064] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:57:08.292] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:57:09.395] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:59:09.903] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 20:59:11.693] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:01:30.605] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:01:32.588] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:02:42.444] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:02:44.193] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:03:36.993] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:03:38.851] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:03:57.109] [ERROR] normal - [TypeError: Cannot read property 'TResult' of undefined]
TypeError: Cannot read property 'TResult' of undefined
    at Answer.json (d:\Workspaces\node\ccorolla\lib\response\answer.js:161:48)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:69:22
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\Q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 21:05:57.407] [ERROR] normal - [TypeError: Cannot read property 'TResult' of undefined]
TypeError: Cannot read property 'TResult' of undefined
    at Answer.json (d:\Workspaces\node\ccorolla\lib\response\answer.js:161:48)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:69:22
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\Q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 21:07:02.639] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:07:04.489] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:07:23.802] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:07:25.729] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:11:59.102] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:12:01.225] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:12:37.056] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:12:38.788] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:12:54.430] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:94:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 21:12:55.457] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:94:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 21:14:07.017] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:14:08.873] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:16:58.694] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:17:00.695] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:55:22.412] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 21:55:23.209] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 22:11:51.142] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 22:11:52.929] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 22:12:42.253] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 22:12:43.874] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 22:19:56.643] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:94:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 22:19:58.142] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:94:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 22:29:11.096] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:94:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-17 23:25:35.288] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:25:37.181] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:29:48.079] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:29:49.596] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:42:06.984] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:42:08.238] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:50:21.565] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:50:23.309] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:51:58.457] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:52:00.325] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:53:53.780] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:53:55.052] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:54:32.095] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-17 23:54:33.935] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 08:15:37.924] [ERROR] normal - checked redis connect is error!
[2015-03-18 08:15:37.941] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-18 08:15:51.391] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 08:15:52.920] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 10:02:30.810] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 10:02:32.158] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:15:35.093] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:15:36.549] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:17:20.934] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:21:51.529] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:21:53.100] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:22:09.895] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:22:11.205] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:23:30.276] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:23:51.229] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:24:22.762] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:24:23.999] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:25:15.981] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:25:17.579] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:26:34.042] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:26:35.697] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:27:11.019] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:27:12.741] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:27:30.217] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:27:31.799] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:28:22.366] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:28:24.077] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:29:48.380] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:29:50.065] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:32:22.792] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:32:24.496] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:33:06.928] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:33:08.334] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:34:21.701] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:34:23.401] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:34:42.334] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:34:43.562] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:34:52.895] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:34:54.365] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:37:38.065] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:37:39.237] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:37:44.715] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:37:45.736] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:38:50.252] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:38:51.732] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:43:15.344] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:43:17.046] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:46:46.353] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:46:47.579] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:47:33.803] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:47:35.098] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:48:32.611] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 15:48:34.310] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:25:11.023] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:25:12.322] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:25:25.713] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:95:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 16:30:04.487] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:30:06.121] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:30:08.190] [ERROR] normal - [TypeError: Cannot read property 'TResult' of undefined]
TypeError: Cannot read property 'TResult' of undefined
    at Answer.json (d:\Workspaces\node\ccorolla\lib\response\answer.js:161:48)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:72:28
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\Q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 16:30:10.601] [ERROR] normal - [TypeError: Cannot read property 'TResult' of undefined]
TypeError: Cannot read property 'TResult' of undefined
    at Answer.json (d:\Workspaces\node\ccorolla\lib\response\answer.js:161:48)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:72:28
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\Q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 16:31:39.286] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:31:40.971] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:31:41.559] [ERROR] normal - [TypeError: Cannot read property 'TResult' of undefined]
TypeError: Cannot read property 'TResult' of undefined
    at Answer.json (d:\Workspaces\node\ccorolla\lib\response\answer.js:161:48)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:73:28
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\Q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\Q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 16:32:19.420] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:32:20.615] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:34:39.137] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:34:40.738] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:35:23.121] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:35:24.203] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:37:43.499] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:37:44.993] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 16:38:10.823] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:95:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 16:38:12.308] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:95:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 17:00:26.873] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 17:00:28.199] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 18:44:30.086] [ERROR] normal - checked redis connect is error!
[2015-03-18 18:44:30.122] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-18 18:44:38.661] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 18:44:40.434] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 18:44:48.210] [ERROR] normal - { [Error: ER_NO_SUCH_TABLE: Table 'ecar.cm_customer' doesn't exist]
  code: 'ER_NO_SUCH_TABLE',
  errno: 1146,
  sqlState: '42S02',
  index: 0 }
Error: ER_NO_SUCH_TABLE: Table 'ecar.cm_customer' doesn't exist
    at Query.Sequence._packetToError (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:82:28)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:748:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:410:10)
    --------------------
    at Protocol._enqueue (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:185:25)
    at Object.select (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:58:10)
    at d:\Workspaces\node\ccorolla\lib\biz\customer.js:16:23
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 18:44:55.288] [ERROR] normal - { [Error: ER_NO_SUCH_TABLE: Table 'ecar.cm_customer' doesn't exist]
  code: 'ER_NO_SUCH_TABLE',
  errno: 1146,
  sqlState: '42S02',
  index: 0 }
Error: ER_NO_SUCH_TABLE: Table 'ecar.cm_customer' doesn't exist
    at Query.Sequence._packetToError (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:82:28)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:748:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:410:10)
    --------------------
    at Protocol._enqueue (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:185:25)
    at Object.query (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:116:10)
    at d:\Workspaces\node\ccorolla\lib\core\business.js:53:24
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 18:44:57.270] [ERROR] normal - { [Error: ER_NO_SUCH_TABLE: Table 'ecar.cm_customer' doesn't exist]
  code: 'ER_NO_SUCH_TABLE',
  errno: 1146,
  sqlState: '42S02',
  index: 0 }
Error: ER_NO_SUCH_TABLE: Table 'ecar.cm_customer' doesn't exist
    at Query.Sequence._packetToError (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:82:28)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:748:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:410:10)
    --------------------
    at Protocol._enqueue (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:185:25)
    at Object.select (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:58:10)
    at d:\Workspaces\node\ccorolla\lib\biz\customer.js:16:23
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 18:45:13.103] [ERROR] normal - { [Error: ER_NO_SUCH_TABLE: Table 'ecar.cm_customer' doesn't exist]
  code: 'ER_NO_SUCH_TABLE',
  errno: 1146,
  sqlState: '42S02',
  index: 0 }
Error: ER_NO_SUCH_TABLE: Table 'ecar.cm_customer' doesn't exist
    at Query.Sequence._packetToError (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:82:28)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:748:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:410:10)
    --------------------
    at Protocol._enqueue (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:185:25)
    at Object.select (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:58:10)
    at d:\Workspaces\node\ccorolla\lib\biz\customer.js:16:23
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 18:45:16.219] [ERROR] normal - { [Error: ER_NO_SUCH_TABLE: Table 'ecar.cm_customer' doesn't exist]
  code: 'ER_NO_SUCH_TABLE',
  errno: 1146,
  sqlState: '42S02',
  index: 0 }
Error: ER_NO_SUCH_TABLE: Table 'ecar.cm_customer' doesn't exist
    at Query.Sequence._packetToError (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:82:28)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:748:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:410:10)
    --------------------
    at Protocol._enqueue (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:185:25)
    at Object.query (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:116:10)
    at d:\Workspaces\node\ccorolla\lib\core\business.js:53:24
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-18 19:04:51.102] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 19:04:52.767] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 19:06:04.383] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 19:06:05.998] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 20:08:45.050] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 20:08:45.687] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 20:22:09.982] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 20:22:11.411] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 20:25:29.567] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 20:25:30.955] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 21:22:25.428] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 21:22:27.121] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 21:24:02.885] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 21:24:04.089] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 21:34:18.654] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 21:34:19.603] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 21:35:01.574] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 21:35:02.877] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 23:30:32.025] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 23:30:33.669] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-18 23:32:07.251] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-18 23:32:08.491] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-19 08:13:23.647] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-19 08:13:25.651] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-22 21:37:39.653] [ERROR] normal - checked redis connect is error!
[2015-03-22 21:37:39.673] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-24 15:03:47.024] [ERROR] normal - checked redis connect is error!
[2015-03-24 15:03:47.043] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-24 15:50:49.727] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 15:50:51.136] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-24 15:55:36.699] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 15:55:38.345] [ERROR] normal - [TypeError: Object function (name){
    var logger = log4js.getLogger(name||'normal');
    //logger.setLevel('INFO');
    return logger;
} has no method 'use']
TypeError: Object function (name){
    var logger = log4js.getLogger(name||'normal');
    //logger.setLevel('INFO');
    return logger;
} has no method 'use'
    at initServer (d:\Workspaces\node\ccorolla\app\app.js:35:27)
    at start (d:\Workspaces\node\ccorolla\main.js:21:33)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-24 15:55:38.352] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-24 15:56:39.550] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 15:56:40.912] [ERROR] normal - [ReferenceError: getLogger is not defined]
ReferenceError: getLogger is not defined
    at Function.exports.use (d:\Workspaces\node\ccorolla\lib\core\logger.js:18:33)
    at initServer (d:\Workspaces\node\ccorolla\app\app.js:35:27)
    at start (d:\Workspaces\node\ccorolla\main.js:21:33)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-24 15:56:40.919] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-24 15:57:08.135] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 15:57:09.795] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-24 15:57:35.338] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 15:57:36.635] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-24 16:16:59.542] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 21:57:35.040] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 21:57:36.762] [ERROR] normal - { [Error: Cannot find module './$global'] code: 'MODULE_NOT_FOUND' }
Error: Cannot find module './$global'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (d:\Workspaces\node\ccorolla\lib\core\business.js:4:13)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
[2015-03-24 22:16:46.183] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 22:16:46.768] [ERROR] normal - { [Error: Cannot find module './$global'] code: 'MODULE_NOT_FOUND' }
Error: Cannot find module './$global'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (d:\Workspaces\node\ccorolla\lib\core\business.js:4:13)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
[2015-03-24 22:17:40.442] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 22:17:42.035] [ERROR] normal - { [Error: Cannot find module './$global'] code: 'MODULE_NOT_FOUND' }
Error: Cannot find module './$global'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (d:\Workspaces\node\ccorolla\lib\core\business.js:4:13)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
[2015-03-24 22:18:24.470] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 22:18:26.138] [ERROR] normal - { [Error: Cannot find module './$global'] code: 'MODULE_NOT_FOUND' }
Error: Cannot find module './$global'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (d:\Workspaces\node\ccorolla\lib\core\sequence.js:4:13)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
[2015-03-24 22:18:53.320] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 22:18:54.969] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-24 22:21:09.712] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 22:21:11.125] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-24 22:39:16.041] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-24 22:39:17.839] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-24 22:43:42.673] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 09:51:20.234] [ERROR] normal - checked redis connect is error!
[2015-03-25 09:51:20.254] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-25 09:51:45.322] [ERROR] normal - checked redis connect is error!
[2015-03-25 09:51:45.339] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-25 09:53:55.148] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 09:53:56.588] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 09:57:02.068] [ERROR] normal - [TypeError: Object #<defer> has no method 'resove']
TypeError: Object #<defer> has no method 'resove'
    at testa (d:\Workspaces\node\ccorolla\main.js:42:15)
    at Object.<anonymous> (d:\Workspaces\node\ccorolla\main.js:52:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
[2015-03-25 09:57:02.082] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 09:57:02.845] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 09:57:57.011] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 09:57:58.135] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 10:45:03.816] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:28:27.803] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:28:27.840] [ERROR] normal - [TypeError: Object [object Promise] has no method 'failure']
TypeError: Object [object Promise] has no method 'failure'
    at d:\Workspaces\node\ccorolla\lib\qb\index.js:44:16
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 11:30:39.341] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:30:39.401] [ERROR] normal - [TypeError: Object function (autoCommit){
    var deferred = Q.defer();
    pool.getConnection(function (err, conn) {
        if (err) {
            return deferred.reject(err);
        }
        conn.autoCommit = autoCommit;
        if (autoCommit) {
            return deferred.resolve(conn);
        }
        conn.beginTransaction(function (err1) {
            if (err1) {
                conn.release();
                return deferred.reject(err);
            }
            deferred.resolve(conn);
        });
    });
    return deferred.promise;
} has no method 'failure']
TypeError: Object function (autoCommit){
    var deferred = Q.defer();
    pool.getConnection(function (err, conn) {
        if (err) {
            return deferred.reject(err);
        }
        conn.autoCommit = autoCommit;
        if (autoCommit) {
            return deferred.resolve(conn);
        }
        conn.beginTransaction(function (err1) {
            if (err1) {
                conn.release();
                return deferred.reject(err);
            }
            deferred.resolve(conn);
        });
    });
    return deferred.promise;
} has no method 'failure'
    at d:\Workspaces\node\ccorolla\lib\qb\index.js:44:16
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 11:30:49.478] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:30:49.512] [ERROR] normal - [TypeError: Object function (autoCommit){
    var deferred = Q.defer();
    pool.getConnection(function (err, conn) {
        if (err) {
            return deferred.reject(err);
        }
        conn.autoCommit = autoCommit;
        if (autoCommit) {
            return deferred.resolve(conn);
        }
        conn.beginTransaction(function (err1) {
            if (err1) {
                conn.release();
                return deferred.reject(err);
            }
            deferred.resolve(conn);
        });
    });
    return deferred.promise;
} has no method 'failure']
TypeError: Object function (autoCommit){
    var deferred = Q.defer();
    pool.getConnection(function (err, conn) {
        if (err) {
            return deferred.reject(err);
        }
        conn.autoCommit = autoCommit;
        if (autoCommit) {
            return deferred.resolve(conn);
        }
        conn.beginTransaction(function (err1) {
            if (err1) {
                conn.release();
                return deferred.reject(err);
            }
            deferred.resolve(conn);
        });
    });
    return deferred.promise;
} has no method 'failure'
    at d:\Workspaces\node\ccorolla\lib\qb\index.js:44:16
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 11:32:38.312] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:32:38.355] [ERROR] normal - [TypeError: Object #<RowDataPacket> has no method 'forEach']
TypeError: Object #<RowDataPacket> has no method 'forEach'
    at setBaseCache (d:\Workspaces\node\ccorolla\lib\biz\base.js:10:11)
    at d:\Workspaces\node\ccorolla\lib\biz\base.js:33:22
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 11:33:12.655] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:33:13.935] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:33:34.078] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:33:35.370] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:34:12.887] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:34:14.519] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:38:23.550] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:38:24.859] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:39:59.819] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 11:40:01.167] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:06:32.115] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:16:56.475] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:16:57.541] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:17:19.493] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:17:20.589] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:17:27.038] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:17:27.995] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:17:41.393] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:17:43.057] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:51:05.750] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 12:51:06.745] [INFO] normal - connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:09:36.909] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:09:59.824] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:10:29.326] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:10:44.599] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:11:24.766] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:11:33.533] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:12:33.300] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:15:18.004] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:16:39.071] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:22:22.174] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:33:57.998] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:36:08.153] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:36:32.303] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:37:07.371] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:37:11.877] [ERROR] normal - [TypeError: Object [object Promise] has no method 'begin']
TypeError: Object [object Promise] has no method 'begin'
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:52:23)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 13:37:11.882] [ERROR] normal - [TypeError: Object [object Promise] has no method 'failure']
TypeError: Object [object Promise] has no method 'failure'
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:77:27
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 13:38:49.891] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:40:42.711] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:41:30.883] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:41:52.394] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:41:57.282] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:63:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 13:42:39.682] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:42:41.566] [ERROR] normal - [ReferenceError: cust2 is not defined]
ReferenceError: cust2 is not defined
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:52:26)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 13:42:51.252] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:42:55.878] [ERROR] normal - [TypeError: Object #<Answer> has no method 'db']
TypeError: Object #<Answer> has no method 'db'
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:74:28
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 13:49:35.572] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:49:39.747] [ERROR] normal - [TypeError: Object #<Answer> has no method 'db']
TypeError: Object #<Answer> has no method 'db'
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:74:28
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 13:49:46.413] [ERROR] normal - [TypeError: Object #<Answer> has no method 'db']
TypeError: Object #<Answer> has no method 'db'
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:74:28
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 13:50:55.750] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:50:59.736] [ERROR] normal - [TypeError: Cannot read property 'TResult' of undefined]
TypeError: Cannot read property 'TResult' of undefined
    at Answer.json (d:\Workspaces\node\ccorolla\lib\response\answer.js:163:48)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:74:18
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 13:52:08.178] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:53:19.301] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:55:19.150] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 13:55:49.331] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:02:33.312] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:03:07.558] [ERROR] normal - [TypeError: Cannot read property 'host' of undefined]
TypeError: Cannot read property 'host' of undefined
    at matrix (d:\Workspaces\node\ccorolla\lib\redis\redis-matrix.js:16:24)
    at Store.expire (d:\Workspaces\node\ccorolla\lib\redis\redis-session.js:44:9)
    at Session.expire (d:\Workspaces\node\ccorolla\lib\redis\session.js:66:22)
    at Session.save (d:\Workspaces\node\ccorolla\lib\redis\session.js:76:20)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:198:41
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
[2015-03-25 14:06:55.676] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:07:04.969] [ERROR] normal - [TypeError: Cannot read property 'host' of undefined]
TypeError: Cannot read property 'host' of undefined
    at getPool (d:\Workspaces\node\ccorolla\lib\redis\redis-matrix.js:70:20)
    at module.exports (d:\Workspaces\node\ccorolla\lib\redis\redis-matrix.js:57:14)
    at Store.expire (d:\Workspaces\node\ccorolla\lib\redis\redis-session.js:44:9)
    at Session.expire (d:\Workspaces\node\ccorolla\lib\redis\session.js:66:22)
    at Session.save (d:\Workspaces\node\ccorolla\lib\redis\session.js:76:20)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:198:41
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
[2015-03-25 14:08:23.522] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:08:32.907] [ERROR] normal - [TypeError: Cannot read property 'host' of undefined]
TypeError: Cannot read property 'host' of undefined
    at getPool (d:\Workspaces\node\ccorolla\lib\redis\redis-matrix.js:70:20)
    at module.exports (d:\Workspaces\node\ccorolla\lib\redis\redis-matrix.js:57:14)
    at Store.save (d:\Workspaces\node\ccorolla\lib\redis\redis-session.js:72:9)
    at Session.save (d:\Workspaces\node\ccorolla\lib\redis\session.js:96:22)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:198:41
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
[2015-03-25 14:14:18.400] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:17:31.907] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:19:54.777] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:21:48.307] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:24:26.847] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:25:10.234] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:25:24.948] [ERROR] normal - [TypeError: Object #<RedisClient> has no method 'releasePool1']
TypeError: Object #<RedisClient> has no method 'releasePool1'
    at d:\Workspaces\node\ccorolla\lib\redis\redis-matrix.js:16:16
    at dispense (d:\Workspaces\node\ccorolla\node_modules\generic-pool\lib\generic-pool.js:250:16)
    at Object.me.acquire (d:\Workspaces\node\ccorolla\node_modules\generic-pool\lib\generic-pool.js:319:5)
    at module.exports (d:\Workspaces\node\ccorolla\lib\redis\redis-matrix.js:13:10)
    at Store.save (d:\Workspaces\node\ccorolla\lib\redis\redis-session.js:71:9)
    at Session.save (d:\Workspaces\node\ccorolla\lib\redis\session.js:96:22)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:198:41
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
[2015-03-25 14:26:32.332] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:26:47.337] [ERROR] normal - [ReferenceError: client is not defined]
ReferenceError: client is not defined
    at d:\Workspaces\node\ccorolla\lib\redis\redis-session.js:83:13
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 14:28:47.629] [ERROR] normal - [ReferenceError: client is not defined]
ReferenceError: client is not defined
    at d:\Workspaces\node\ccorolla\lib\redis\redis-session.js:83:13
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 14:36:36.844] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:36:49.802] [ERROR] normal - [TypeError: Cannot read property 'releasePool1' of undefined]
TypeError: Cannot read property 'releasePool1' of undefined
    at d:\Workspaces\node\ccorolla\lib\redis\redis-session.js:29:19
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 14:37:31.375] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 14:40:54.588] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 15:01:40.332] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 15:01:54.559] [ERROR] normal - [Error: too fast,have a rest!]
Error: too fast,have a rest!
    at Q.all.spread.then.then.done.self.json.PWD (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:167:32)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:02:00.892] [ERROR] normal - [Error: too fast,have a rest!]
Error: too fast,have a rest!
    at Q.all.spread.then.then.done.self.json.PWD (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:167:32)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:02:04.909] [ERROR] normal - [Error: too fast,have a rest!]
Error: too fast,have a rest!
    at Q.all.spread.then.then.done.self.json.PWD (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:167:32)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:03:00.128] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:60:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:03:02.270] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:60:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:03:12.796] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:60:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:06:00.291] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 15:06:08.068] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:61:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:08:11.879] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 15:08:13.884] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:61:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:08:49.992] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 15:08:53.509] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:62:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:09:13.360] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 15:09:17.520] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:62:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:09:41.924] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 15:09:43.646] [ERROR] normal - [TypeError: Cannot set property 'CUST_ID' of undefined]
TypeError: Cannot set property 'CUST_ID' of undefined
    at Q.all.spread.then.then.then.then.then.qb.remove.table (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:62:25)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 15:10:31.840] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 16:08:44.863] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 19:14:00.318] [ERROR] normal - checked redis connect is error!
[2015-03-25 19:14:00.338] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-25 19:14:08.833] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 19:16:30.865] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 19:17:55.792] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 19:19:01.560] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:20:28.242] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:22:06.715] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:27:29.451] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:28:01.301] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:29:01.800] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:29:08.773] [ERROR] normal - [TypeError: Cannot call method 'toUpperCase' of undefined]
TypeError: Cannot call method 'toUpperCase' of undefined
    at Object.table (d:\Workspaces\node\ccorolla\lib\qb\partials.js:56:43)
    at Function.exports.select (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:52:21)
    at d:\Workspaces\node\ccorolla\lib\qb\index.js:65:19
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:30:31.349] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:31:45.265] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:31:51.555] [ERROR] normal - [ReferenceError: rs is not defined]
ReferenceError: rs is not defined
    at d:\Workspaces\node\ccorolla\lib\qb\index.js:67:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:32:09.054] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:32:19.141] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:118:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:54:41.150] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:54:52.526] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:118:34)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:55:10.076] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:55:15.370] [ERROR] normal - [TypeError: Object Error: dd has no method 'query']
TypeError: Object Error: dd has no method 'query'
    at Function.exports.insert (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:74:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:135:23
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:55:15.379] [ERROR] normal - [TypeError: Object Error: dd has no method 'rollback']
TypeError: Object Error: dd has no method 'rollback'
    at Function.exports.failure (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:35:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:145:27
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:57:47.318] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:57:51.476] [ERROR] normal - [TypeError: Object Error: 13588348096 already exists! has no method 'query']
TypeError: Object Error: 13588348096 already exists! has no method 'query'
    at Function.exports.insert (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:74:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:133:23
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:57:51.486] [ERROR] normal - [TypeError: Object Error: 13588348096 already exists! has no method 'rollback']
TypeError: Object Error: 13588348096 already exists! has no method 'rollback'
    at Function.exports.failure (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:35:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:143:29
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:59:08.707] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:59:11.768] [ERROR] normal - [TypeError: Object Error: 13588348096 already exists! has no method 'query']
TypeError: Object Error: 13588348096 already exists! has no method 'query'
    at Function.exports.insert (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:74:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:133:23
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:59:11.775] [ERROR] normal - [TypeError: Object Error: 13588348096 already exists! has no method 'rollback']
TypeError: Object Error: 13588348096 already exists! has no method 'rollback'
    at Function.exports.failure (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:35:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:144:29
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:59:22.412] [ERROR] normal - [TypeError: Object Error: 13588348096 already exists! has no method 'query']
TypeError: Object Error: 13588348096 already exists! has no method 'query'
    at Function.exports.insert (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:74:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:133:23
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:59:22.414] [ERROR] normal - [TypeError: Object Error: 13588348096 already exists! has no method 'rollback']
TypeError: Object Error: 13588348096 already exists! has no method 'rollback'
    at Function.exports.failure (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:35:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:144:29
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:59:42.205] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 20:59:45.377] [ERROR] normal - [TypeError: Object Error: 13588348096 already exists! has no method 'query']
TypeError: Object Error: 13588348096 already exists! has no method 'query'
    at Function.exports.insert (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:74:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:133:23
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 20:59:45.383] [ERROR] normal - [TypeError: Object Error: 13588348096 already exists! has no method 'rollback']
TypeError: Object Error: 13588348096 already exists! has no method 'rollback'
    at Function.exports.failure (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:35:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:144:29
    at _rejected (d:\Workspaces\node\ccorolla\node_modules\q\q.js:804:24)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:830:30
    at Promise.when (d:\Workspaces\node\ccorolla\node_modules\q\q.js:1064:31)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:748:41)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 21:09:43.344] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 21:09:55.039] [ERROR] normal - [Error: 13588348096 already exists!]
Error: 13588348096 already exists!
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$cust-info.js:132:42)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 21:11:20.395] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 21:11:26.093] [ERROR] normal - 13588348096 already exists!
[2015-03-25 21:15:50.230] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 21:15:56.354] [ERROR] normal - 13588348096 already exists!
[2015-03-25 21:26:34.309] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 21:31:17.463] [ERROR] normal - 13588348096 already exists!
[2015-03-25 21:31:23.634] [ERROR] normal - 13588348096 already exists!
[2015-03-25 21:31:44.576] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 21:33:29.115] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 21:33:48.989] [ERROR] normal - 13588348096 already exists!
[2015-03-25 21:34:34.561] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 21:34:37.333] [ERROR] normal - 13588348096 already exists!
[2015-03-25 21:35:25.164] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 21:35:34.805] [ERROR] normal - 13588348096 already exists!
[2015-03-25 21:36:24.969] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 21:36:44.829] [ERROR] normal - 13588348096 already exists!
[2015-03-25 21:36:56.109] [ERROR] normal - { [Error: ER_DATA_TOO_LONG: Data too long for column 'cust_id' at row 1]
  code: 'ER_DATA_TOO_LONG',
  errno: 1406,
  sqlState: '22001',
  index: 0 }
Error: ER_DATA_TOO_LONG: Data too long for column 'cust_id' at row 1
    at Query.Sequence._packetToError (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:82:28)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:748:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:410:10)
    --------------------
    at Protocol._enqueue (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:185:25)
    at Function.exports.insert (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:74:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:134:23
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 21:58:14.530] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 21:58:25.327] [ERROR] normal - { [Error: ER_DATA_TOO_LONG: Data too long for column 'cust_last_done' at row 1]
  code: 'ER_DATA_TOO_LONG',
  errno: 1406,
  sqlState: '22001',
  index: 0 }
Error: ER_DATA_TOO_LONG: Data too long for column 'cust_last_done' at row 1
    at Query.Sequence._packetToError (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:82:28)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:748:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:410:10)
    --------------------
    at Protocol._enqueue (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:185:25)
    at Function.exports.insert (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:74:10)
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:134:23
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-25 22:00:21.423] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 22:04:53.134] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-25 22:52:27.413] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 08:14:47.073] [ERROR] normal - checked redis connect is error!
[2015-03-26 08:14:47.090] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-26 08:15:00.284] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 12:47:32.155] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 12:47:47.681] [ERROR] normal - 13588348096 already exists!
[2015-03-26 15:16:19.154] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 15:29:58.207] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 15:54:04.677] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 20:11:53.977] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 20:14:04.606] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 20:14:26.923] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 21:01:07.930] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:02:10.271] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:10:32.242] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:27:10.156] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:27:31.826] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:33:09.428] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:33:59.283] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:34:20.503] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:34:41.093] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:35:03.219] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:35:23.250] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:35:52.131] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:37:27.088] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:42:46.576] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-26 22:43:15.524] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 12:15:45.938] [ERROR] normal - checked redis connect is error!
[2015-03-27 12:15:45.957] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-27 12:15:52.474] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 12:18:34.812] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 12:20:06.787] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 12:20:47.642] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 12:29:59.236] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 12:30:28.904] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 13:35:02.125] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 13:53:44.488] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 13:54:43.074] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 13:55:34.085] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 14:14:21.693] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 14:20:38.008] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 14:34:19.485] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:22:02.143] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:22:33.905] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:23:18.904] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:27:55.691] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:28:59.331] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:29:22.035] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:30:22.726] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:30:53.860] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:31:35.928] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:33:01.488] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:35:09.368] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:35:50.412] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:36:28.419] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:36:57.607] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:37:41.166] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:39:00.135] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 16:47:07.691] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 20:05:24.623] [ERROR] normal - checked redis connect is error!
[2015-03-27 20:05:24.640] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-27 20:05:34.027] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 20:05:40.125] [ERROR] normal - [TypeError: Cannot call method 'call' of undefined]
TypeError: Cannot call method 'call' of undefined
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:173:27
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-27 20:05:41.232] [ERROR] normal - [TypeError: Cannot call method 'call' of undefined]
TypeError: Cannot call method 'call' of undefined
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:173:27
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-27 20:05:47.226] [ERROR] normal - [TypeError: Cannot call method 'call' of undefined]
TypeError: Cannot call method 'call' of undefined
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:173:27
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-27 20:05:48.569] [ERROR] normal - [TypeError: Cannot call method 'call' of undefined]
TypeError: Cannot call method 'call' of undefined
    at d:\Workspaces\node\ccorolla\app\car\$cust-info.js:173:27
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-27 20:06:49.769] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 20:12:16.157] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 20:12:23.438] [ERROR] normal - 13588348096 already exists!
[2015-03-27 21:00:38.978] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 21:00:43.182] [ERROR] normal - 13588348096 already exists!
[2015-03-27 21:01:09.565] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-27 21:01:53.961] [ERROR] normal - 13588348096 already exists!
[2015-03-28 20:47:41.446] [ERROR] normal - checked redis connect is error!
[2015-03-28 20:47:41.453] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-28 20:47:58.527] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-28 20:48:04.006] [ERROR] normal - 13588348096 already exists!
[2015-03-28 20:50:37.298] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-28 20:50:43.252] [ERROR] normal - 13588348096 already exists!
[2015-03-28 20:51:04.541] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-28 20:51:08.266] [ERROR] normal - 13588348096 already exists!
[2015-03-28 21:57:33.245] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 12:22:41.177] [ERROR] normal - checked redis connect is error!
[2015-03-29 12:22:41.194] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-29 12:22:50.290] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 12:23:16.413] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 12:23:55.125] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 17:02:22.843] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 18:50:58.693] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 18:51:09.773] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 18:57:01.453] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 18:58:10.883] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 19:04:51.045] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 19:06:30.888] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 19:10:03.788] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 19:10:35.535] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 19:12:03.554] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 19:24:53.808] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 20:02:57.704] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 21:09:21.035] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 21:09:21.403] [ERROR] normal - [SyntaxError: Unexpected token }]
SyntaxError: Unexpected token }
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at d:\Workspaces\node\ccorolla\lib\core\$include.js:56:21
    at Array.forEach (native)
    at Object.module.exports [as $include] (d:\Workspaces\node\ccorolla\lib\core\$include.js:54:14)
    at initServer (d:\Workspaces\node\ccorolla\app\app.js:12:37)
[2015-03-29 21:09:55.546] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 21:10:02.057] [ERROR] normal - 13588348096 already exists!
[2015-03-29 21:21:07.354] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 21:21:16.270] [ERROR] normal - undefined already exists!
[2015-03-29 21:25:10.608] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 21:25:14.244] [ERROR] normal - { [Error: ER_BAD_FIELD_ERROR: Unknown column 'CUST_ID' in 'field list']
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlState: '42S22',
  index: 0 }
Error: ER_BAD_FIELD_ERROR: Unknown column 'CUST_ID' in 'field list'
    at Query.Sequence._packetToError (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:82:28)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:748:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:410:10)
    --------------------
    at Protocol._enqueue (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (d:\Workspaces\node\ccorolla\node_modules\mysql\lib\Connection.js:185:25)
    at Function.exports.select (d:\Workspaces\node\ccorolla\lib\qb\sql-execute.js:58:10)
    at qb.then.then.v (d:\Workspaces\node\ccorolla\lib\qb\index.js:66:19)
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-29 21:25:57.469] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 21:25:59.518] [ERROR] normal - undefined already exists!
[2015-03-29 21:25:59.684] [ERROR] normal - undefined already exists!
[2015-03-29 21:26:20.427] [ERROR] normal - undefined already exists!
[2015-03-29 21:26:21.420] [ERROR] normal - undefined already exists!
[2015-03-29 21:27:08.536] [ERROR] normal - undefined already exists!
[2015-03-29 21:27:20.274] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 21:27:22.497] [ERROR] normal - undefined already exists!
[2015-03-29 21:27:23.337] [ERROR] normal - undefined already exists!
[2015-03-29 21:30:47.790] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 21:30:51.143] [ERROR] normal - undefined already exists!
[2015-03-29 21:31:09.616] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 21:31:13.444] [ERROR] normal - undefined already exists!
[2015-03-29 22:07:10.954] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 23:40:33.508] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 23:40:39.467] [ERROR] normal - [TypeError: Cannot read property 'CREATE_DATE' of undefined]
TypeError: Cannot read property 'CREATE_DATE' of undefined
    at d:\Workspaces\node\ccorolla\app\car\$car-query.js:150:35
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-29 23:40:39.473] [ERROR] normal - [ReferenceError: actor is not defined]
ReferenceError: actor is not defined
    at d:\Workspaces\node\ccorolla\app\car\$car-query.js:97:25
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-29 23:44:45.642] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 23:44:51.548] [ERROR] normal - [ReferenceError: actor is not defined]
ReferenceError: actor is not defined
    at d:\Workspaces\node\ccorolla\app\car\$car-query.js:97:25
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-29 23:44:51.557] [ERROR] normal - [ReferenceError: actor is not defined]
ReferenceError: actor is not defined
    at d:\Workspaces\node\ccorolla\app\car\$car-query.js:151:38
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-29 23:45:43.787] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 23:45:49.540] [ERROR] normal - [ReferenceError: actor is not defined]
ReferenceError: actor is not defined
    at d:\Workspaces\node\ccorolla\app\car\$car-query.js:97:25
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:564:44
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-29 23:47:56.047] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 23:48:55.367] [ERROR] normal - [ReferenceError: carinfo is not defined]
ReferenceError: carinfo is not defined
    at Object.exports.needApproveIllegal (d:\Workspaces\node\ccorolla\app\car\mgr\cust-busi-mgr.js:88:29)
    at Q.all.spread.then.conn (d:\Workspaces\node\ccorolla\app\car\$car-query.js:123:25)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:1171:26
    at _fulfilled (d:\Workspaces\node\ccorolla\node_modules\q\q.js:794:54)
    at self.promiseDispatch.done (d:\Workspaces\node\ccorolla\node_modules\q\q.js:823:30)
    at Promise.promise.promiseDispatch (d:\Workspaces\node\ccorolla\node_modules\q\q.js:756:13)
    at d:\Workspaces\node\ccorolla\node_modules\q\q.js:516:49
    at flush (d:\Workspaces\node\ccorolla\node_modules\q\q.js:110:17)
    at process._tickCallback (node.js:419:13)
[2015-03-29 23:49:13.306] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-29 23:54:50.834] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 13:07:45.226] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 13:57:38.406] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 13:59:27.461] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 16:51:25.646] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 16:51:26.053] [ERROR] normal - [ReferenceError: illegalqueryByCust is not defined]
ReferenceError: illegalqueryByCust is not defined
    at d:\Workspaces\node\ccorolla\app\car\$car-query.js:151:13
    at Object.<anonymous> (d:\Workspaces\node\ccorolla\app\car\$car-query.js:153:2)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at d:\Workspaces\node\ccorolla\lib\core\$include.js:56:21
    at Array.forEach (native)
[2015-03-30 16:51:43.674] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 16:54:13.911] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 17:00:27.821] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 20:46:41.285] [ERROR] normal - checked redis connect is error!
[2015-03-30 20:46:41.303] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\Workspaces\node\ccorolla\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:95:17)
    at net.js:440:14
    at process._tickCallback (node.js:419:13)
[2015-03-30 20:47:06.431] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 21:09:16.024] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 21:20:07.558] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-03-30 21:47:35.819] [INFO] normal - checked, connect  to redis address [localhost:6379]is ok!
[2015-05-28 11:10:11.862] [ERROR] normal - checked redis connect is error!
[2015-05-28 11:10:11.887] [ERROR] normal - [Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED]
Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED
    at RedisClient.on_error (d:\nodeTest\node_modules\redis\index.js:196:24)
    at Socket.<anonymous> (d:\nodeTest\node_modules\redis\index.js:106:14)
    at Socket.emit (events.js:107:17)
    at net.js:459:14
    at process._tickCallback (node.js:355:11)
